apps/catalogue/src/views/VariableExplorer.vue

```diff
@@ -18,6 +18,7 @@
             v-model="keywords"
             :isMultiSelect="true"
             tableName="Keywords"
+            schemaName="CatalogueOntologies"
```

Comment:
same is this an application / module constant ?

Reply:
yes, same one as above